src/Menu/Menu.tsx

```diff
@@ -34,43 +34,53 @@ export const MenuNavigator = () => {
 export const Menu = () => {
   const strings = useStrings()
   const navigateTo = useNavigateTo()
-  const walletMetas = useWalletMetas()
-  const walletCount = walletMetas?.length || ''
 
   return (
     <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.root}>
       <ScrollView contentContainerStyle={styles.scrollViewContent} bounces={false}>
         <AppSettings //
           label={strings.appSettings}
           onPress={navigateTo.appSettings}
-          left={<Icon.Gear size={24} color="#6B7384" />}
+          left={<Icon.Gear size={24} color={lightPalette.gray['600']} />}
         />
 
-        <HR />
+        <Hr />
 
-        <AllWallets
-          label={`${strings.allWallets} (${walletCount})`}
-          onPress={navigateTo.allWallets}
-          left={<Icon.Wallets size={24} color="#6B7384" />}
+        {/* <Releases //
```

Comment:
@stackchain 
my preference is to not leave in commentted code
if its part of another feature it can be included in another branch
or saved in a jira ticked

Reply:
thoughts?

Reply:
let's do it, @jemubm would let's keep it out (move to Jira) till we can merge it, perhaps on Monday during the planning we can decide to go/no-go with that for 4.8

Reply:
we need to cover 2 things

1. context for privacy mode
2. flow to the wallet settings while in the transition

Reply:
These are the tickets created:

- https://emurgo.atlassian.net/browse/YOMO-281
- https://emurgo.atlassian.net/browse/YOMO-283
- https://emurgo.atlassian.net/browse/YOMO-276

Reply:
The code commented has been removed.